caliban/config.py

```diff
@@ -78,6 +78,33 @@ class JobMode(str, Enum):
 }
 
 
+def extract_dlvm_image(job_mode: JobMode, dlvm_arg: str) -> str:
+  """Returns the DLVM image url for the job model and the comand line arg
+  or returns None if the key doesn't exist in the config.
+
+  """
+  return _dlvm_config(job_mode).get(dlvm_arg)
+
+
+def _dlvm_config(job_mode: JobMode) -> Dict[str, str]:
```

Comment:
nice, this is good. I think I see now why you have `--dlvm` vs `--base_image`. What we COULD do is have a prefix for `--base_image`, something like `--base_image dlvm:pytorch`, that would force a lookup here. That would let this feature give us general base images too.